### PR TITLE
[build] Set main stack size from prj.conf instead of prj.mdef

### PR DESCRIPTION
**Pull request by grgustaf**
Signed-off-by: Geoff Gustafson <geoff@linux.intel.com>

### CODE REVIEW

**Comment by jimmy-huang** [comment]
+1

File: Makefile [hunk 1]
```diff
@@ -173,7 +173,6 @@ analyze: $(JS)
 ifeq ($(DEV), ashell)
 	@cat fragments/prj.mdef.dev >> prj.mdef
```
Comment (jimmy-huang) [inline Makefile]: shouldn't prj.mdef.dev and prj.mdef.heap  be removed as well? or move to prj.conf equivalent?

Comment (poussa) [inline Makefile]: yes, we don't need to .mdef file at all. The heap pool can be defined in .conf file as well (CONFIG_HEAP_MEM_POOL_SIZE)

Comment (grgustaf) [inline Makefile]: Patches accepted. :)

I wanted to commit the part that I had figured out so far to make some progress...

**Comment by grgustaf** [comment]
Added a patch to remove heap size setting and a couple more of the files, but there is more to do to completely eliminate mdef. I think we can get these in though.

**Comment by jimmy-huang** [comment]
+1

**Pull request merged**